src/button/config.js

```diff
@@ -241,6 +241,40 @@ export const BUTTON_CONFIG : ButtonConfig = {
         allowPrimaryVertical:   false,
         allowPrimaryHorizontal: true
     },
+    [ BUTTON_LABEL.ITAU ]: {
+        label:     `{ logo: ${ BUTTON_LOGO.ITAU } }`,
+        logoLabel: `{ logo: ${ BUTTON_LOGO.ITAU } }`,
+
+        defaultColor: BUTTON_COLOR.SILVER,
+
+        colors: [
+            BUTTON_COLOR.BLUE,
+            BUTTON_COLOR.SILVER,
+            BUTTON_COLOR.BLACK,
+            BUTTON_COLOR.WHITE
+        ],
+
+        logoColors:  {
+            [ BUTTON_COLOR.BLUE ]:   BUTTON_LOGO_COLOR.WHITE,
```

Comment:
If they're the same color, then let's pick either white or blue and change this config to match.